.travis.yml

```diff
@@ -15,7 +15,9 @@ cache:
     - docs/node_modules
 before_script: npm prune
 
-script: npm run test-with-coverage
+script:
+  - npm run flow:check
+  - npm run test-with-coverage
```

Comment:
Wow. So I will prefer a script that run flow type depending on the platform so when I run "npm test", I don't miss flow check.